spec/io/io_spec.rb

```diff
@@ -34,6 +34,16 @@
       end
     end
 
+    context ".from_csv_url" do
+      it "loads from a CSV file" do
+        df = Daru::DataFrame.from_csv_url('https://raw.githubusercontent.com/anshuman23/ruBayes/master/test/sample.csv')
```

Comment:
I have made all the other changes, but this is the only thing I'm a little bit unsure of in terms of implementation.

I know how to use webmock and vcr together to create an offline response, however, I'm not sure where I will 'require' all the modules (vcr and webmock here) as all the test files essentially cannot have modules being called inside (?)
This is also related to another problem - where do I write the initial configuration for the vcr cassette being used? 

If you could give me some insight, I'll make the required additions ASAP